src/main/scala/org/dbpedia/spotlight/wikistats/JsonPediaParser.scala

```diff
@@ -112,17 +154,44 @@ class JsonPediaParser(inputWikiDump:String, lang:String)
    */
   def getSfURI(): RDD[(Long,String,String)]= {
 
-    val sfUriRDD = dfWikiRDD.select("wid","links.description","links.id")
+    /*
```

Comment:
Sure. That was the initial logic which I had which was kind of complicated...so commented it to use explode method of dataframe.